src/util/XMLUtils.cpp

```diff
@@ -109,18 +122,25 @@ bool XMLUtils::GetBoolean(const TiXmlNode* pRootNode, const char* strTag, bool&
   return true;
 }
 
-bool XMLUtils::GetString(const TiXmlNode* pRootNode, const char* strTag, CStdString& strStringValue)
+bool XMLUtils::GetString(const TiXmlNode* pRootNode, const char* strTag, std::string& strStringValue)
 {
-  const TiXmlElement* pElement = pRootNode->FirstChildElement(strTag );
+  const TiXmlElement* pElement = pRootNode->FirstChildElement(strTag);
   if (!pElement) return false;
   const TiXmlNode* pNode = pElement->FirstChild();
   if (pNode != NULL)
   {
-    strStringValue = pNode->Value();
+    strStringValue = pNode->ValueStr();
     return true;
   }
-  strStringValue.Empty();
-  return false;
+  strStringValue.clear();
+  return true;
+}
+
+std::string XMLUtils::GetString(const TiXmlNode* pRootNode, const char* strTag)
```

Comment:
The name of the PR is misleading. The commit msg is `Sync XMLUtils with kodi master` and that's what I've done. As far as I remember I haven't added a single line myself.